src/engine/enginebuffer.cpp

```diff
@@ -689,7 +696,12 @@ void EngineBuffer::doSeekPlayPos(double new_playpos, enum SeekRequest seekType)
     }
 #endif
 
-    queueNewPlaypos(new_playpos, seekType);
+    queueNewPlaypos(position, seekType);
+}
+
+void EngineBuffer::doSeekPlayPos(double new_playpos, enum SeekRequest seekType) {
```

Comment:
I suggest to rename this method explicitly to `doSeekPlayPosLegacy`/`doSeekPlayPosSamples`/`doSeekPlaySamplePos`(?) instead of using function overloading.

Reply:
I wanted to keep the changes in calling code to a minimum to reduce the size of the diff. Once we migrated all the engine control classes, we should just remove it.

Reply:
Nevermind, it's a private method. I removed it.